packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/StyledMenuItemBase.tsx

```diff
@@ -38,7 +38,7 @@ export const StyledMenuItemBase = styled.li<MenuItemBaseProps>`
     switch (accent) {
       case 'danger': {
         return css`
-          color: ${theme.font.color.danger};
+          color: ${theme.color.red};
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below